SpatialGDK/Source/SpatialGDK/Private/SpatialGDKModule.cpp

```diff
@@ -2,6 +2,11 @@
 
 #include "SpatialGDKModule.h"
 
+// clang-format off
+#include "SpatialConstants.h"
+#include "SpatialConstants.cxx"
```

Comment:
Is it worth just making this an implementation file if somewhere in the project we need to include it only once?

Reply:
What do you mean by "making it an implementation file" ?

Reply:
As in compiled and linked separately (instead of adding symbols to the *Module.cpp file), although a compiled constants file does seem a bit odd

Reply:
Oh, you mean a static lib. Yeah, that's what it should be, but I'm not sure if I want to fiddle with the build system to achieve that.

Reply:
Ah right yeah I see why it was done this way now - I agree

I wonder if it could be made more explicit like with comment + #if GDK_CONSTANTS_IMPL which includes the file from SpatialConstants.h (if we ever find ourselves creating a new module from scratch)